etc/test-helper.js

```diff
@@ -1,11 +1,3 @@
-var validationErrorFormatter = require('./validation-error-message-formatter.js');
```

Comment:
This change is not directly related to the primary purpose of this pull request, but it only involves moving this code further down the file so it seemed unnecessary to create a new issue and PR just for it.